tests/app/shared/issue-tables/issues-table-paginated-data.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { MatPaginator, MatPaginatorIntl } from "@angular/material";
+import { Issue } from "../../../../src/app/core/models/issue.model";
+import { TEAM_4 } from "../../../constants/data.constants";
+import {
+    ISSUE_WITH_ASSIGNEES,
+    ISSUE_WITH_EMPTY_DESCRIPTION,
+    ISSUE_WITH_EMPTY_DESCRIPTION_HIGH_SEVERITY,
+    ISSUE_WITH_EMPTY_DESCRIPTION_LOW_SEVERITY
+} from "../../../constants/githubissue.constants";
+import { setAndGetPaginatedData } from '../../../../src/app/shared/issue-tables/issue-tables-paginated-data';
+
+describe('isses-table-paginated-data', () => {
+    describe('getPaginatedData()', () => {
+        const changeDetectorRefMock = {
+            detectChanges: () => null,
+            markForCheck: () => null,
+            detach: () => null,
+            reattach: () => null,
+            checkNoChanges: () => null
+        };
+        let dataSet_3: Issue[];
```

Comment:
Just a small nitpick, but I would like to give a suggestion here: 

```typescript 
const dataSet_3: Issue[] = [ ... ];
``` 

Given we aren't changing the data sets, would it be better to leave them as constants here? :O

Reply:
This is cause during tests the data sets are actually spliced, which modifies the actual data sets! I need to reset the data set before every test

Reply:
Ah okok! 👍